src/web/wms/src/main/java/org/geoserver/wms/web/data/StylePage.java

```diff
@@ -63,6 +65,18 @@ protected Component getComponentForProperty(
                                             id,
                                             StyleProvider.CREATED_TIMESTAMP.getModel(itemModel));
                                 }
+                                if (property == StyleProvider.FORMAT) {
+                                    Version formatVersion =
```

Comment:
Setting up the label is split among the code here and the decated StyleFormatLabel component. Please move everything inside StyleFormatLabel instead, you could give it the entire itemModel as constructor argument for example, and perform extraction and formatting in there.